docs/offline-support.adoc

```diff
@@ -0,0 +1,218 @@
+ifdef::env-github[]
+// If on GitHub, define attributes so we can find our diagram files and render
+// them.
+
+// The branch will be used to find the correct diagrams to render below.
+// When PRing changes to the diagrams you can change this attributes
+// temporarily to the name of the branch you're working on. But don't forget
+// to change it back to main before merging!!
+:github-branch: add-offline-docs
+
+:github-repo: nhsuk/record-childrens-vaccinations
+
+// URL for PlantUML Proxy. Using an attribute mainly because it's just tidier.
+:plantuml-proxy-url: http://www.plantuml.com/plantuml/proxy?cache=no&src=
+
+// Full path prefix we'll use for diagrams below.
+:diagram-path-url: {plantuml-proxy-url}https://raw.githubusercontent.com/{github-repo}/{github-branch}/docs
+endif::[]
+
+:toc:
+
+= Offline support
+
+A key feature required for the Record children's vaccines service is to allow
+clinical users to use parts of the system while not connected to the Internet.
+There are certain clinical settings where Internet access isn't available, for
+example in schools where there may be no mobile signal or WiFi that the SAIS
+team can use.
+
+To accomplish this the standard patterns provided by server-rendered apps are
+insufficient. Aggressive caching can help accommodate viewing data, but updating
+records requires a more advanced approach.
+
+== Feature overview
+
+The current design of the service expects the SAIS team to save a campaign for
```

Comment:
It might be good to explicitly mention here that the data would be stored encrypted.